dev/package-lock.json

```diff
@@ -27,357 +27,215 @@
       "dev": true
     },
     "@fortawesome/fontawesome-common-types": {
-      "version": "0.2.2",
-      "resolved": "https://registry.npmjs.org/@fortawesome/fontawesome-common-types/-/fontawesome-common-types-0.2.2.tgz",
-      "integrity": "sha512-OBzXzYJ+nDMece2nKeuV7hLuwqPN3jv1Fk56gag2DL4BiwWl8gkFQplj5Krep9HwtjFxbM6/DWN6ZG7FRC5N0w=="
+      "version": "0.2.6",
```

Comment:
Delete this from repo, and list in .gitignore!
No need to version lock files when developing